cpp/set_property_module/set_property_module.cpp

```diff
@@ -0,0 +1,296 @@
+#include <map>
+#include <unordered_set>
+
+#include <fmt/core.h>
+#include <mgp.hpp>
+
+#include "mgclient.hpp"
+
+constexpr static std::string_view kResult = "result";
+constexpr static std::string_view kQueryExecuted = "queryExecuted";
+constexpr static std::string_view kSourceProperties = "sourceProperties";
+constexpr static std::string_view kSourceVariable = "sourceVariable";
+constexpr static std::string_view kSourceNode = "sourceNode";
+constexpr static std::string_view kSourceRel = "sourceRel";
+constexpr static std::string_view kTargetProperties = "targetProperties";
+constexpr static std::string_view kTargetVariable = "targetVariable";
+constexpr static std::string_view kTargetNode = "targetNode";
+constexpr static std::string_view kTargetRel = "targetRel";
+
+constexpr char *kMgHost = "MG_HOST";
+constexpr char *kMgPort = "MG_PORT";
+constexpr char *kMgUsername = "MG_USERNAME";
+constexpr char *kMgPassword = "MG_PASSWORD";
+
+constexpr char *kDefaultHost = "localhost";
+constexpr uint16_t kDefaultPort = 7687;
+
+mg::Client::Params GetClientParams() {
+  auto *host = kDefaultHost;
+  auto port = kDefaultPort;
+  auto *username = "";
+  auto *password = "";
```

Comment:
You can just init struct here mg::Client::Params{...} and then update struct element if you find env variable